triagebot.toml

```diff
@@ -583,11 +583,11 @@ message = "Third-party dependency whitelist may have been modified! You must ens
 cc = ["@davidtwco", "@wesleywiser"]
 
 [mentions."src/bootstrap/src/core/config"]
-message = "This PR modifies `src/bootstrap/src/core/config`. If appropriate, please also update `CONFIG_CHANGE_HISTORY` in `src/bootstrap/src/lib.rs` and `change-id` in `config.example.toml`."
+message = "This PR modifies `src/bootstrap/src/core/config`. If appropriate, please provide an explanation of the modification in the PR description, update `CONFIG_CHANGE_HISTORY` in `src/bootstrap/src/lib.rs` and `change-id` in `config.example.toml`."
```

Comment:
It's not about an explanation of the modification. It's about explaining *what rustc devs should change in their config.toml*. Keep in mind who these messages are addressing.

Reply:
Also this has some overlap with https://github.com/rust-lang/rust/pull/117809